gc/structs/ForwardedHeader.hpp

```diff
@@ -40,6 +42,8 @@
    This hint is not necessary for correctness of copying protocol, it's just an optimization to avoid visiting destination object header
    in cases when it's likely not in data cash (GC thread encountering already forwarded object) */
 #define OMR_BEING_COPIED_HINT 2
+/* combine OMR_FORWARDED_TAG with OMR_BEING_COPIED_HINT into one mask which should be stripped from the pointer in order to remove all tags */
+#define ALL_FORWARDED_TAG (OMR_FORWARDED_TAG | OMR_BEING_COPIED_HINT)
```

Comment:
It should also include OMR_SELF_FORWARDED_TAG (0x1). Then ALL tag will really mean all 3 lowest possible bits of the source object forwarding address. 
About naming:
- prepend OMR_ to it
- i know that ALL came from old OpenJ9 tag name, but consider using MASK instead of ALL (it's more common in this file, and elsewhere in GC code): maybe OMR_FORWARDED_TAG_MASK?